pkg/oc/cli/cmd/process.go

```diff
@@ -88,10 +88,7 @@ func NewCmdProcess(fullName string, f *clientcmd.Factory, in io.Reader, out, err
 	}
 	cmd.Flags().StringP("filename", "f", "", "Filename or URL to file to read a template")
 	cmd.MarkFlagFilename("filename", "yaml", "yml", "json")
-	params := cmd.Flags().StringArrayP("value", "v", nil, "Specify a key-value pair (eg. -v FOO=BAR) to set/override a parameter value in the template.")
```

Comment:
This was deprecated in 1.5 (over a year ago) so I think we're safe entirely dropping it.

Reply:
> This was deprecated in 1.5 (over a year ago) so I think we're safe entirely dropping it.

@liggitt you're more conservative than me.  Arguments against?

Reply:
> Short flags conflict with long flags? I didn't expect that.

huh.  I didn't realize that it registered `--v` and -v`.  That's suprising.

Reply:
drop it

Reply:
Yeah, there's a `AddGoFlags` call, which adds a shorthand `-v` for `--v`. So for now we can't have that shorthand. We only can use the slightly longer one, but slowly getting there ;)